watertap/unit_models/anaerobic_digestor.py

```diff
@@ -884,11 +884,11 @@ def initialize_build(
             optarg = {}
 
         # Check DOF
-        if degrees_of_freedom(self) != 0:
-            raise InitializationError(
-                f"{self.name} degrees of freedom were not 0 at the beginning "
-                f"of initialization. DoF = {degrees_of_freedom(self)}"
-            )
+        # if degrees_of_freedom(self) != 0:
+        #     raise InitializationError(
+        #         f"{self.name} degrees of freedom were not 0 at the beginning "
+        #         f"of initialization. DoF = {degrees_of_freedom(self)}"
+        #     )
```

Comment:
Similarly, I made tweaks in #948 that should be useful for the AD unit initialization (just a start though). Furthermore, we can add a try/except to the sensitivity run so that we simply allow the solve to continue even after a failed init,

Reply:
Yes, the main issue here is that for the initialization procedure I used the model has 0 DOF, even thought the reactor still has some DOF. And this if statement makes the flowsheet fail. I was considering adding an option in the configuration to accommodate for both